src/components/LeaderBoard/Leaderboard.jsx

```diff
@@ -261,18 +262,42 @@ const LeaderBoard = ({
                   </div>
 
                   {/* <Link to={`/dashboard/${item.personId}`}> */}
-                  <div
-                    title={`Weekly Committed: ${item.weeklycommittedHours} hours`}
-                    style={{
-                      backgroundColor:
-                        item.tangibletime >= item.weeklycommittedHours ? 'green' : 'red',
-                      width: 15,
-                      height: 15,
-                      borderRadius: 7.5,
-                      margin: 'auto',
-                      verticalAlign: 'middle',
-                    }}
-                  />
+                  {
+                    hasLeaderboardPermissions(loggedInUser.role) && 
+                    item.weeklycommittedHours !== 0 && item.tangibletime >= item.weeklycommittedHours * 1.25 ? (
+                        <i
+                        className="fa fa-star"
+                        title={`Weekly Committed: ${item.weeklycommittedHours} hours`}
+                        style={{
+                          color:
+                          item.tangibletime >= item.weeklycommittedHours * 1.75
+                              ? 'purple'
+                              : item.tangibletime >= item.weeklycommittedHours * 1.5 &&
+                              item.tangibletime < item.weeklycommittedHours * 1.75
+                              ? 'fuchsia'
+                              : item.tangibletime >= item.weeklycommittedHours * 1.25 &&
+                              item.tangibletime < item.weeklycommittedHours * 1.5
+                              ? 'darkgreen'
+                              : null,
+                          fontSize: '20px',
+                          marginLeft: '16px',
+                          verticalAlign: 'middle',
```

Comment:
How about change this to:

remove marginLeft 

add

display: 'flex', 
alignItems: 'center', 
justifyContent: 'center', 

![Screenshot 2023-05-30 11-10-12](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/105796041/e32d28f6-11ed-4074-b3b1-3c0393511468)

Reply:
Will do! Thanks for the suggestion @eduhorta